packages/app/package.json

```diff
@@ -37,8 +37,9 @@
     "@backstage/plugin-techdocs-react": "^1.2.9-next.2",
     "@backstage/plugin-user-settings": "^0.8.14-next.2",
     "@backstage/theme": "^0.6.0-next.1",
-    "@material-ui/core": "^4.11.0",
-    "@material-ui/icons": "^4.9.1",
+    "@mui/icons-material": "5.14.18",
+    "@mui/material": "5.14.18",
+    "@mui/styles": "5.14.18",
```

Comment:
Is there's a better way to keep this in sync with Backstage itself I'm very much open to ideas.

Reply:
https://github.com/backstage/backstage/blob/446001ee4348020051ca554e4918806efb972bd0/packages/theme/package.json#L45

Reply:
@webark curious if you have any suggestions here based on you experience?

Reply:
We set a resolution in the root.

Reply:
It may be the case that we could revisit these being peer dependencies, like we do for react, react-dom etc. I.e. that all Backstage packages put them as peer deps and then it's up to the adopter to fulfil those in their packages/app to a concrete version. But I'm not sure it's the perfect solution, would have to bring it up with the team